api/proto/teleport/userpreferences/v1/unified_tab.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.userpreferences.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/userpreferences/v1;userpreferencesv1";
+
+// UnifiedTab is the selected default tab in the unified resources view
+enum UnifiedTab {
```

Comment:
I like the rename. I think we should not combine `card` and `table` but it makes sense to make it object like so that its easy to add that because its coming up soon anyway. Thanks

Reply:
This was a great idea, thank you. I didn't add in the card/table stuff yet since thats not included but we can add a new field like `DefaultView` when we add it, and I've changed this field to be named `DefaultTab`. Then just whatever combination of both will end up being what the web ui shows (they can be discrete from each other).